### DISCUSSION TITLE
Is there a project roadmap?

### DISCUSSION THREAD

**Discussion by xLinkOut** [ellite/Wallos#279] [General]
HI! I recently discovered Wallos and, after using it for a while, I've identified a few features that I'd like to see implemented (for example, more notification providers like Telegram). I also took a look at the issues and it seems that a refactor of the notification module is underway. I would like to know if there is some sort of roadmap or something similar for the project, so as to know what it is possible to contribute and how.

Thank you!

**Comment by ellite** [comment]
Hi. No roadmap.
But I currently have two ongoing feature branches. 
One with themes and one with more notification methods.
Telegram and webhooks are the new methods I'm trying to implement.
I still have to figure out how the telegram notifications work, and that's why is taking so long.

**Reply by xLinkOut** [reply]
Okay, thanks a lot! What branch are you working on Telegram notifications in? I have already worked in the past and also recently with Telegram bots, I could possibly give you a hand in my free time. (Maybe I'll start with translation in my native language (Italian))

**Reply by ellite** [reply]
It's still only a local branch.
I can push it later.

The Italian translation would be very much appreciated.

**Reply by gitandriu** [reply]
Maybe Apprise is a good option for notifications. Provides a common framework compatible with many popular services (including Telegram), so more flexible than limiting just to Telegram.

https://github.com/caronc/apprise

**Reply by ellite** [reply]
Both features are live.
